_includes/2017_data/header-home.html

```diff
@@ -19,7 +19,9 @@
                         <a href="/"></a>
                     </li>
                     <li>
-                        <a href="{{prepend: site.baseurl }}/cfp">CFP</a>
+            <!-- Hard coding because I cant figure out the base url and its late                        -->
+            <!--                        <a href="{{prepend: site.baseurl}}/cfp">CFP</a>-->
+                        <a href="https://pymc-devs.github.io/pymcon/cfp">CFP</a>
```

Comment:
I think that should be `{{ '/cfp' | relative_url }}` from https://talk.jekyllrb.com/t/relative-url-and-baseurl/2051/6 cc @sidravi1 @canyon289

Reply:
see here too though https://github.com/pymc-devs/pymcon/pull/19#discussion_r464063908